scala/private/rule_impls.bzl

```diff
@@ -700,16 +712,18 @@ def scala_test_impl(ctx):
                       jars.transitive_compile_jars, jars.jars2labels)
   # _scalatest is an http_jar, so its compile jar is run through ijar
   # however, contains macros, so need to handle separately
-  scalatest_jars = collect_jars([ctx.attr._scalatest]).transitive_runtime_jars
+  scalatest_jars = collect_jars(
+      ctx.attr._scalatest).transitive_runtime_jars
   cjars = depset(transitive = [cjars, scalatest_jars])
   transitive_rjars = depset(transitive = [transitive_rjars, scalatest_jars])
 
   if is_dependency_analyzer_on(ctx):
     transitive_compile_jars = depset(
         transitive = [scalatest_jars, transitive_compile_jars])
     scalatest_jars_list = scalatest_jars.to_list()
-    add_labels_of_jars_to(jars_to_labels, ctx.attr._scalatest,
-                          scalatest_jars_list, scalatest_jars_list)
+    for dep in ctx.attr._scalatest:
```

Comment:
As scalatest 3.0.5 requires scalactic I changed `ctx.attr._scalatest` to a `label_list` containing both scalatest and scalactic jars. This might interfere with how the scalatest version is overridden when using bazel-deps.

In `WORKSPACE`:
```
bind(
    name = "io_bazel_rules_scala/dependency/scalatest/scalatest",
    actual = "//3rdparty/jvm/org/scalatest",
)
```
and
```
  org.scalatest:
    scalatest:
      exports:
        - "org.scalactic:scalactic"
      lang: scala
      version: "3.0.4"
```

Not sure how to combine two jars under one label in the workspace